src/index.tsx

```diff
@@ -62,76 +48,34 @@ export const Zage = ({
     return <></>;
   }
   return (
-    <View
-      style={{
-        backgroundColor: 'transparent',
-        position: 'absolute',
-        bottom: 0,
-        top: 0,
-        left: 0,
-        right: 0,
-        zIndex: 99999999,
-      }}
+    <Modal
+      visible={showZage}
+      transparent={false}
+      animationType='none'
+      presentationStyle='overFullScreen'
+      style={{ backgroundColor: 'transparent' }}
     >
-      <WebView
-        source={{ uri: zageApp }}
-        javaScriptEnabled={true}
-        nativeID={'zageWebview'}
-        pagingEnabled={true}
-        javaScriptCanOpenWindowsAutomatically={true}
-        injectedJavaScript={jsResponse}
-        useSharedProcessPool={false}
-        automaticallyAdjustsScrollIndicatorInsets={true}
-        contentInsetAdjustmentBehavior={'automatic'}
-        limitsNavigationsToAppBoundDomains={false}
-        onShouldStartLoadWithRequest={(request) => {
-          const params = getParams(request.url);
-          if (oAuth) return false;
-          const cleanedUrl = request.url.toLowerCase();
-          const isOAuthInstitution = oAuthUrls
-            .map((url) => cleanedUrl.includes(url))
-            .some((url) => url);
-          const needsInAppBrowser = isOAuthInstitution || params?.['redirect_uri'];
-          if (needsInAppBrowser) {
-            InAppBrowser.close();
-            setOAuth(true);
-            InAppBrowser.open(request.url, {
-              // iOS Properties
-              dismissButtonStyle: 'done',
-              animated: true,
-              ephemeralWebSession: true,
-              modalPresentationStyle: 'formSheet',
-              modalEnabled: true,
-              modalTransitionStyle: 'coverVertical',
-              // Android Properties
-              showTitle: true,
-              enableUrlBarHiding: true,
-              enableDefaultShare: true,
-              forceCloseOnRedirection: true,
-            })
-              .then(() => {
-                setOAuth(false);
-              })
-              .catch(() => {
-                setOAuth(false);
-              });
-            return false;
-          }
-          return true;
-        }}
-        style={{ backgroundColor: 'transparent' }}
-        onMessage={(event) => {
-          const res = JSON.parse(event.nativeEvent.data);
-          if (res.completed) {
-            setShowZage(false);
-            onComplete(res.response);
-          } else {
+      <SafeAreaView style={{ flex: 0, backgroundColor: '#1F2937', opacity: 0.75 }} />
```

Comment:
I should've thought of this, this is a genius fix: just need to remember to change it in one spot if we change it in the other